### ISSUE TITLE
PrintStack to false as default

### ISSUE THREAD

**Issue by bragagia** [urfave/negroni#214]
`PrintStack` should be set to `false` as default in `negroni.Recovery`.
It should be set willingly to true only in a protected development environment but never in production.

**Comment by jszwedko** [comment]
Hi @baijum,

While I agree with you that `false` makes a more sensible default for this, unfortunately this default has existed since the `Recovery` middleware was introduced in 6939988 so hesitate to break compatibility and change it.

If there is ever a v2 of negroni, this should definitely be updated. I'll mark this as future for now.

**Comment by erikacarvalho** [comment]
hey @streppel check this out 🤓

**Comment by Solaris-star** [comment]
Fair point for production safety, but changing the default would be a breaking change for existing users who rely on stack traces in logs. Since this project follows semver, it would need a major version bump.

A middle ground: keep `PrintStack: true` as default but add a prominent note in the README recommending `PrintStack: false` for production. The `Recovery` docs already show how to configure it.
